charts/flux2/templates/helm-controller.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if and .Values.helmcontroller.create}}
+apiVersion: v1
```

Comment:
can they be separated to have a single manifest template?

Reply:
yes lets do it in a followup PR - thanks